src/js/components/Form/Form.js

```diff
@@ -90,7 +91,7 @@ const Form = forwardRef(
           setErrors(prevErrors => {
             const nextErrors = { ...prevErrors };
             Object.keys(prevErrors).forEach(errName => {
-              if (validations.current[errName]) {
+              if (validations.current[errName] && errName === name) {
```

Comment:
I'm OK with this, since it works.

But, it might be cleaner to:
```
if (prevErrors[name] && validations.current[name]) {
  const nextErrors = { ...prevErrors };
  ...
}
```
rather than looping though all `prevErrors`.

Reply:
changes made